src/app/graphql.module.ts

```diff
@@ -1,23 +1,27 @@
-import {NgModule} from '@angular/core';
-import {APOLLO_OPTIONS} from 'apollo-angular';
-import {ApolloClientOptions, InMemoryCache} from '@apollo/client/core';
-import {HttpLink} from 'apollo-angular/http';
+import { NgModule } from '@angular/core';
+import { Apollo } from 'apollo-angular';
+import { InMemoryCache } from '@apollo/client/core';
+import {
+  HttpBatchLink,
+  HttpBatchLinkModule,
+} from 'apollo-angular-link-http-batch';
 
 const uri = 'https://axieinfinity.com/graphql-server-v2/graphql'; // <-- add the URL of the GraphQL server here
-export function createApollo(httpLink: HttpLink): ApolloClientOptions<any> {
-  return {
-    link: httpLink.create({uri}),
-    cache: new InMemoryCache(),
-  };
-}
+// export function createApollo(httpLink: HttpBatchLink) {
+//   return {
+//     link: httpLink.create({ uri, batchMax: 20 }),
+//     cache: new InMemoryCache(),
+//   };
+// }
```

Comment:
It's cool how this does all the batching in the background :), nice finding. Also don't be afraid to remove uncommented code.

Reply:
awesome, I have removed those and the your TODO on other file